src/modules/appbuilder.js

```diff
@@ -0,0 +1,55 @@
+import Template from './obarr.js';
+import localGet from './localstorage';
+
+const todoListContainer = document.querySelector('.listItem');
+
+const displayList = () => {
+  todoListContainer.innerHTML = '';
+  localGet.forEach((item, id) => {
+    todoListContainer.innerHTML
+    += `
+      <div class='toDoItem'>
+        <input class='item' id='check-${id}',
+         completed)' type='checkbox' value=${item.completed}>
+        <input type='text' class='findInput' id='input-${id}' value=${item.description} />
+        <i onclick='updateList(${id})' class='fa fas fa-check' id='options-${id}'></i>
+        <i onclick='removeList(${id})' class='fas fa-trash' id='delete-${id}'></i>
+      </div>
+    `;
+  });
+};
```

Comment:
- [ ] As an example: when I add a task with the description "Read a book", your UI displays a task with the description "task" instead of "Read a book". Please ensure the correct description is displayed on the UI.
- [ ] When the user reloads the page, edits, or removes a task, all the tasks' descriptions on the UI are changed to "task". Kindly fix that.

<img width="267" alt="image" src="https://user-images.githubusercontent.com/77190203/194644596-53230d00-f188-4332-b15f-0bde07425ab0.png">

Reply:
![image](https://user-images.githubusercontent.com/100999396/194648925-9b493258-5191-459c-bc3b-3ecd71d56ef6.png)

@medaminedev66 

thanks for the reviewer, this is how it displays from on my system.

Reply:
done